source/bao_hyp/config.rst

```diff
@@ -483,6 +483,16 @@ where:
     - ``iommu.fq_irq_id`` [mandatory if IOMMU is available] - the Fault Queue interrupt ID (the
       current implementation assumes this is a wired interrupt);
 
+.. note::
+
+   When mapping MMIO regions for guests, the memory regions associated with the GIC (Generic
+   Interrupt Controller) must be excluded. Mapping these regions can lead to conflicts or incorrect
+   behavior, as they are typically managed by Bao through trap-and-emulate mechanisms.
```

Comment:
When mapping MMIO regions for guests, the memory regions associated with the interrupt controller (only interrupt controllers are target of MMIO trap and emulate in Bao) must be excluded. Mapping these regions can lead to conflicts or incorrect
   behavior, as they are typically managed by Bao through trap-and-emulate mechanisms.